docs/release-notes.md

```diff
@@ -11,14 +11,35 @@ We are continually making changes in order to improve and grow the Red Hat Desig
 <nav class="releases-links">
   <rh-block id="changelog-block">
     <a href="https://github.com/RedHat-UX/red-hat-design-system/releases">
-      <h2>Changelog</h2>
+      Changelog
     </a>
   </rh-block>
 </nav>
 
 <section class="release-versions">
 <section class="section release-version">
 
+## Version 1.2.0
+Released October 16, 2023
+
+### Highlights
+
+| Change                         | Notes {style="width: 70%" } |
+| ------------------------------ | --------------------------------- |
+| Added `<rh-table>`             | A table is a container for displaying information. It allows a user to scan, examine, and compare large amounts of data. |
+| Added `<rh-tile>`              | A tile is a flexible layout with a clickable and contained surface. |
+| Added `<rh-timestamp>`         | Provides consistent formats for displaying date and time values. |
+| Added `<rh-navigation-secondary>` current page indicator support | Updated support for a current page indicator using `aria-current="page"`. |
+| Improved keyboard navigation on `<rh-navigation-secondary>` | Secondary Navigation now has improved keyboard navigation. |
+| Fixed `<rh-card>` `header` slot | Card's header slot now displays items vertically instead of stacking, allowing for more than one item to display in the header. |
+| Fixed `<rh-cta>` `brick` variant | Brick variants of calls-to-action (CTAs) are now full width. |
```

Comment:
"Calls to action" isn't hyphenated in the docs.

Reply:
Done